readme.md

```diff
@@ -3,8 +3,8 @@
 This is the parent project of Einstein Bot Channel Connector. This contains following submodules:
 
 * [channel-connector-starter](channel-connector-starter) : Channel Connector Framework is spring boot starter that simplifies building channel connectors for the salesforce einstein bot.
```

Comment:
Can you also update README under `channel-connector-starter` with correct references to `ui-connector`?

Specifically these lines

> See [EinsteinBotController](https://github.com/forcedotcom/einstein-bot-channel-connector/blob/master/channel-connector-example/src/main/java/com/salesforce/einsteinbot/connector/example/EinsteinBotController.java) for example code and [channel-connector-example](https://github.com/forcedotcom/einstein-bot-channel-connector/blob/master/channel-connector-example)[](https://github.com/forcedotcom/einstein-bot-channel-connector/tree/master/channel-connector-starter#observability) dir fully working application.

> See [NewRelicMetricsExportAutoConfiguration](https://github.com/forcedotcom/einstein-bot-channel-connector/blob/master/channel-connector-example/src/main/java/com/salesforce/einsteinbot/connector/example/NewRelicMetricsExportAutoConfiguration.java) as example code for publishing to New Relic.

Reply:
Sure, will do that.